src/orquestra/integrations/cirq/conversions/_openfermion_conversions.py

```diff
@@ -1,8 +1,13 @@
 ################################################################################
 # © Copyright 2021-2022 Zapata Computing Inc.
 ################################################################################
-from openfermion import QubitOperator, SymbolicOperator  # type: ignore
-from orquestra.quantum.operators import PauliRepresentation, PauliSum, PauliTerm
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    # Numpy throws deprecation warnings due to the scipy import
```

Comment:
Is there any way to upgrade numpy? Or is that going to be a lot of work?

Reply:
This issue comes from actually numpy being too high versionwise. 1.25 numpy throws deprecation warnings on import scipy import, so the only way would be to restrict numpy which is so widely used across almost every single library that I wouldnt wont to do it tbh